src/panel/direction/RouteSummaryInfo.jsx

```diff
@@ -5,17 +5,17 @@ import RouteVia from './RouteVia';
 import { Badge } from 'src/components/ui';
 import { formatDuration, formatDistance } from 'src/libs/route_utils';
 
-const RouteSummaryInfo = ({ isFastest, route, vehicle }) =>
+const RouteSummaryInfo = ({ isFastest, route, vehicle, showDistance = false }) =>
   <div>
     <div className="u-text--title u-mb-xxxs route-summary-info-duration">
       {formatDuration(route.duration)}
     </div>
 
     <RouteVia className="u-mb-xs" route={route} vehicle={vehicle} />
 
-    <Badge className="u-mr-xs">
+    {showDistance && <Badge className="u-mr-xs">
```

Comment:
I think spacing utils have been removed ?

Reply:
Not yet, they are still useful until we switch to the real design system :)